### PR TITLE
Bpk 817 deprecate headings and paragraphs

### PR DESCRIPTION
**Pull request by k0nserv**
This removes the usage of `BpkHeading` and `BpkParagraph` in the docs and as part of stories.

### CODE REVIEW

File: packages/bpk-docs/src/components/Heading/Heading.scss [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+@import '~bpk-mixins/index';
+
+.bpk-docs-heading {
+  margin-top: 0;
+  margin-bottom: $bpk-spacing-sm;
+
+  &--h1,
+  &--h2,
+  &--h3 {
+    font-weight: 400;
```
Comment (k0nserv) [inline packages/bpk-docs/src/components/Heading/Heading.scss]: Good point

**Comment by k0nserv** [comment]
This also regress mobile heading responsiveness. Do we wanna use `bpk-component-breakpoint` to achieve that or just a CSS media query?

**Comment by matthewdavidson** [comment]
> This also regress mobile heading responsiveness. Do we wanna use bpk-component-breakpoint to achieve that or just a CSS media query?

I like the declarative nature of using `BpkBreakpoint`, but behind the scenes it uses the `window.matchMedia` api which I suspect isn't as performant as plain ole' media queries. 

@tvooo, thoughts?

**Comment by tvooo** [comment]
Agree with both points. Like using BpkBreakpoint, but it's probably not as performant. Then again, how often does the window size change?

**Comment by tvooo** [comment]
Happy with the changes. I can imagine that the duplicate mixins will confuse people, not sure if we can/should do something about that?

**Comment by matthewdavidson** [comment]
> Happy with the changes. I can imagine that the duplicate mixins will confuse people, not sure if we can/should do something about that?

Yeah Im not particularly happy with them either - i guess you understand why i did it though?

**Comment by tvooo** [comment]
Yep sure. Otherwise I'd have "requested changes" :trollface: Go merge!

**Pull request merged**